Firestore/core/src/firebase/firestore/nanopb/byte_string.cc

```diff
@@ -91,6 +92,16 @@ std::string ByteString::ToString() const {
   return absl::StrCat("<", hex, ">");
 }
 
+std::string ByteString::ToHumanReadableString() const {
```

Comment:
Feel free to push back on this. I added it to help with debugging, but then I thought it could be useful in the future.

Reply:
I'm fine with swapping which one is the default. I've changed it to:
- use C-style escape sequences for some of the more common control characters;
- use the hexadecimal representation of the character (`\\0x0b`) for other control characters.

I'm a little doubtful about the second one because this wouldn't produce a valid C string (all characters after `\0x` are supposed to be a part of the hexadecimal number, not just the first two). Let me know if you have a better idea.

Reply:
This should now be handled by `absl::CEscape`, which uses octal sequences.